### DISCUSSION TITLE
关于企业空间角色自动更新问题

### DISCUSSION THREAD

**Discussion by Jimyhjy** [kubesphere/kubesphere#6124] [Q&A]
为什么系统中没有外部触发 workspacerole CR的更新，但系统还是会不定时通过workspacerole_controller调谐 更新CR数据，求帮忙解答
![image](https://github.com/kubesphere/kubesphere/assets/14359380/49bc5ae0-eb72-43e2-9b43-3a57d9aac36c)

**Comment by calvinyv** [comment]
which version you are using?
these is a issue related to the sync latency for workspace and already fixed in latest version
